### PR TITLE
nova: fix order placement

### PR DESCRIPTION
**Pull request by Tristian**

### CODE REVIEW

**Comment by crackfoo** [comment]
I merged this already and it looks like it's working good now. trades executed and also withdraw requested automatically as well.

@Tristian thanks.

ps feel free to drop by #yiimp :) I've always got little ideas and projects to do on yiimp :)

lemme know what addy to send to for payment.

Cheers

**Comment by Tristian** [comment]
Excellent :)
I didn't know #yiimp was a thing. I'll drop in.
1KyEKpfRsVKRXX3P3YvwCSdoc3h6tqVk3E

File: web/yaamp/core/trading/nova_trading.php [hunk 1]
```diff
@@ -95,11 +95,10 @@ function doNovaTrading($quick=false)
 		if(!$coin || is_array($coin) || $coin->dontsell) continue;
 
 		sleep(1);
-		$ticker = nova_api_query("market/info/{$pair}");
+		$res = nova_api_query("market/info/{$pair}");
+		$ticket = $res->markets[0];
```
Comment (tpruvot) [inline web/yaamp/core/trading/nova_trading.php]: typo "ticket" and the is_object($res) should be checked

File: web/yaamp/core/trading/nova_trading.php [hunk 2]
```diff
@@ -184,8 +183,10 @@ function doNovaTrading($quick=false)
 
 		sleep(1);
 
-		$pair = "{$symbol}_BTC";
-		$ticker = nova_api_query("market/info/{$pair}");
+		$pair = "BTC_{$symbol}";
+		$info = nova_api_query("market/info/{$pair}");
+		$ticker = $info->markets[0];
```
Comment (tpruvot) [inline web/yaamp/core/trading/nova_trading.php]: same here

**Comment by tpruvot** [comment]
There is also a thing about deposit addresses, symbol2 seems not checked

**Comment by Tristian** [comment]
Do you mean the deposit address update in markets.php? I can't see where any of the other markets check symbol2 either.

**Comment by tpruvot** [comment]
$symbol = $coin->symbol;
                if (!empty($coin->symbol2)) $symbol = $coin->symbol2;

**Pull request merged**